lib/utils/property-order.js

```diff
@@ -81,6 +83,12 @@ function determinePropertyType(node, parentType) {
   }
 
   if (ember.isFunctionExpression(node.value)) {
+    if (node.value.body && node.value.body.body) {
+      if (node.value.body.body.length > 0) {
```

Comment:
Could we merge this condition with the one above and avoid having duplicate `'method'` returns?
Like
```
if (node.value.body && node.value.body.body && node.value.body.body.length === 0) {
  return 'empty-method';
}
```
And since the condition becomes a bit long because of this, maybe put it into a utility function (like `isEmptyMethod`).